commands/verify.js

```diff
@@ -96,25 +55,32 @@ async function getDetails(message,username) {
       let realName = author_detail[i][1];
 
 			let part = realName.split(" ")
+			let q=0, lastName="abc";
+			while(part[q]!== undefined){
+				lastName = part[q];
+				q++;
+			}
 
   //    console.log(`real name ${realName}`);
       let rollNumber = author_detail[i][2];
 		//	console.log(`roll no ${rollNumber}`);
 			let branch = rollNumber.slice(4,6).toLowerCase();
 		//	console.log(`branch ${branch}`);
 			let year = rollNumber.slice(0,2);
+			let gradYear = parseInt(year);
+			gradYear+=4;
 		//	console.log(`year ${year}`);
 			 try {
   			 if (!message.guild.me.hasPermission('MANAGE_NICKNAMES')) return message.reply('I\'m missing permissions.');
   			 if (message.author.id === message.guild.ownerID) return message.reply('I can\'t change your nickname.');
 
-  			await message.member.setNickname(`${username}-${part[0]}`);
-				message.reply("Nickname changed");
+					 await message.member.setNickname(`${username}-${part[0]} ${lastName}`);
+	 				 message.reply("Nickname changed");
```

Comment:
Reply to a message only once in the entire function. You can concatenate the message strings.
msgString += "Nickname changed\n"